user_class/user.rb

```diff
@@ -1,25 +1,24 @@
+# frozen_string_literal: true
+
+# User class
```

Comment:
This comment is not necessary. This type of messages are only helpful if you explain some complex behaviour, if not, you can skip them.